source/_integrations/dio_chacon.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Dio Chacon
+description: Instructions on how to integrate your Dio Chacon devices within Home Assistant.
+ha_category:
+  - Cover
+ha_release: 2024.5
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@cnico'
+ha_domain: dio_chacon
+ha_platforms:
+  - cover
+ha_integration_type: integration
+---
+
+[Dio Chacon devices](https://chacon.com/en/) are connected home devices that can be controlled via RF 433 Mhz or Wifi.
+This integration gives you access to the Wifi connection that allows in Home Assistant to list your devices and interact with them in real time like vendor's smartphone application does.
```

Comment:
```suggestion
This {% term integrations %} gives you access to the Wi-Fi connection so that Home Assistant can list your Chacon Dio devices and interact with them in real time, the same way the vendor's smartphone application does.
```